src/TableSchema.php

```diff
@@ -13,8 +13,14 @@
  */
 final class TableSchema extends AbstractTableSchema
 {
+    private ?string $comment = null;
     private array $foreignKeys = [];
 
+    public function getComment(): ?string
```

Comment:
Suggest add this method to table schema in `yiisoft/db` that return null by default.

Reply:
I prefer to do it in every implementation.

Reply:
But in application I'm work with `TableSchema` from `yiisoft/db` and IDE don't see this methods.

Reply:
It should be part of the interface, not a specific method of implementations.